src/guardian/keys-validation/keys.fixtures.ts

```diff
@@ -1,21 +1,21 @@
 export const validKeys = [
   {
-    key: '0xa9bfaa8207ee6c78644c079ffc91b6e5abcc5eede1b7a06abb8fb40e490a75ea269c178dd524b65185299d2bbd2eb7b2',
+    key: '0xa4d381739a4cc9554bf01c49e827a22ae99d429a79bd74ecfa86b72210c151644e511ce1c5fa4e5fb8d355dec35239e2',
     depositSignature:
-      '0xaa5f2a1053ba7d197495df44d4a32b7ae10265cf9e38560a16b782978c0a24271a113c9538453b7e45f35cb64c7adb460d7a9fe8c8ce6b8c80ca42fd5c48e180c73fc08f7d35ba32e39f32c902fd333faf47611827f0b7813f11c4c518dd2e59',
+      '0xac50577d80539bf0a9ac0ea98d7a98e4bb3c644c28d53c57204c297081cbef7ca47975a2fffc05b873b406e3f08b4b6902e57c61b0d98dc7eac49d677c82a5c4f695232158360c7595c4414f5f27c9a7ab1bbdbafa4f85c967f82a4f68cb6f5e',
     operatorIndex: 1,
-    used: false,
-    moduleAddress: '0x9D4AF1Ee19Dad8857db3a45B0374c81c8A1C6320',
-    index: 51,
+    used: true,
+    moduleAddress: '0x55032650b14df07b85bF18A3a3eC8E0Af2e028d5',
+    index: 0,
   },
   {
-    key: '0xb3c90525010a5710d43acbea46047fc37ed55306d032527fa15dd7e8cd8a9a5fa490347cc5fce59936fb8300683cd9f3',
+    key: '0xa9e4c3d9b71b82ae78da55a686208bb2b6b0b31f7a100f2d9ea46beb86088432dc3d320ccebadef9563e8be4c6ad8e63',
     depositSignature:
-      '0x8a77d9411781360cc107344a99f6660b206d2c708ae7fa35565b76ec661a0b86b6c78f5b5691d2cf469c27d0655dfc6311451a9e0501f3c19c6f7e35a770d1a908bfec7cba2e07339dc633b8b6626216ce76ec0fa48ee56aaaf2f9dc7ccb2fe2',
+      '0xadfcf17804e128039df67d7ecf9f4312bbc17eacd57c76539a4d8cbb7e02cba246083200dd95bd5a83fdfcbcbf7051001432f2aef4e867f9cb426e850744a66b8842ebdb8d22d26bf7531ebdcc72e7dbae608fd4c5dde4a7bf43e65aff002c37',
     operatorIndex: 1,
-    used: false,
-    moduleAddress: '0x9D4AF1Ee19Dad8857db3a45B0374c81c8A1C6320',
-    index: 52,
+    used: true,
```

Comment:
Sorry, I didn't understand why we change fixtures for the unit test of validation. Also, we validate vetted unused keys, which doesn't influence the algorithm, but it's a bit confusing